practice/src/css/sections/best-place.css

```diff
@@ -1 +1,280 @@
+.place-section {
+  background-image: url(https://s3-alpha-sig.figma.com/img/6a46/9b14/71c2ad353c582a71eba56ed3e06dae89?Expires=1710115200&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=f1e04y7hunRZwMK3kfR6sbxf9I5gFjfbaaFA4A1ivKZoaHUUqdroajaBOdrVJz0Hk9J77gTwKBaz~NjoW4nAUg7e0QfE6cWD-wHnoxx9gA8qS4fiiZK3Jnxl2YHiXqGkyI2mWF4ob0IXKO0WCGbe435gNKWJrLgq0Ehc8daojGHU5vnUFn3mRiI-qnEvh3rIuR3aBcqDKXHi7SOI7LPkaC79tz7oRpBZ-kx5KhSipWT2fTuglaheljGHaatDelrIUwgwFhktHi0NiZSFSd0Vx94bQ1ScS-CRYVCK1pfwRBxQ1kNANwmVOEvIHCP8oVy26FQg8Tf~b0Q8i9hBB-LdTA__);
+  background-color: var(--secondary-background-color);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 350px;
+  border-radius: 24px;
+}
 
+.place-introduction {
+  display: none;
+}
+
+.tab-nav {
+  position: absolute;
+  top: 400px;
+}
+
+.tab-list {
+  display: flex;
+  margin: 0;
+  padding-left: 0px;
+  width: fit-content;
+}
+
+.tab-pane {
+  background-color: var(--primary-background-color);
+  border-radius: 0px 20px 20px 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 8px;
+  padding: 15px 10px 10px 10px;
+}
+
+.tab-item {
+  padding: 8px 20px;
+  border-radius: 0px 12px 0px 0px;
+  background-color: var(--primary-background-color);
+}
+
+.native {
+  border-radius: 12px 0px 0px 0px;
+  background-color: var(--tertiary-background-color);
+}
+
+.tab-link {
+  font-weight: var(--font-weight-medium);
+  font-size: var(--font-size-xxsmall);
+  line-height: 18px;
+  font-family: 'Poppins';
+  color: var(--secondary-text-color);
+}
+
+.active {
+  color: white;
+}
+
+.pane-1 {
+  display: flex;
+  gap: 5px;
+  align-items: center;
+}
+
+.pane-content {
+  display: flex;
+  gap: 4px;
+}
+
+.tab-introduction {
+  display: flex;
+  flex-direction: column;
+}
+/*  */
+.tab-title {

```

Comment:
Please help me check the font family for text, because remember that we don't use variable for font family in @font-face